kde-apps/akregator/akregator-9999.ebuild

```diff
@@ -19,39 +19,32 @@ KEYWORDS=""
 IUSE="speech telemetry"
 
 RDEPEND="
-	>=dev-libs/ktextaddons-1.4.42:6
-	>=dev-qt/qtbase-${QTMIN}:6[dbus,gui,network,widgets,xml]
-	>=dev-qt/qtwebengine-${QTMIN}:6
+	>=dev-libs/ktextaddons-1.4.42:6[speech?]
+	>=dev-qt/qtbase-${QTMIN}:6[gui,widgets]
+	>=dev-qt/qtwebengine-${QTMIN}:6[widgets]
 	>=kde-apps/grantleetheme-${PVCUT}:6
 	>=kde-apps/kontactinterface-${PVCUT}:6
-	>=kde-apps/kpimtextedit-${PVCUT}:6[speech=]
 	>=kde-apps/libkdepim-${PVCUT}:6
 	>=kde-apps/messagelib-${PVCUT}:6
 	>=kde-apps/pimcommon-${PVCUT}:6
 	>=kde-frameworks/kcmutils-${KFMIN}:6
 	>=kde-frameworks/kcodecs-${KFMIN}:6
-	>=kde-frameworks/kcompletion-${KFMIN}:6
-	>=kde-frameworks/kconfig-${KFMIN}:6
-	>=kde-frameworks/kconfigwidgets-${KFMIN}:6
```

Comment:
`libKF6ConfigWidgets.so.6 => /usr/lib64/libKF6ConfigWidgets.so.6`

Reply:
If `kde-frameworks/kconfigwidgets:6` is depcleaned for some reason, will it be a missing symbol?